workflow/controller/operator_test.go

```diff
@@ -5897,6 +5897,7 @@ spec:
 		return node.Phase == wfv1.NodePending
 	}))
 
+	time.Sleep(time.Second)
```

Comment:
?

Reply:
the `TestResubmitPendingPods` test flakes occasionally,
tested with `go test -count=50 -run=TestResubmitPendingPods ./workflow/controller`

Seems `deletePods` and `makePodsPhase` uses podInformer, and sometimes they execute too quickly before receiving the pod information